src/confluent_kafka/src/confluent_kafka.c

```diff
@@ -1842,6 +1808,31 @@ int Handle_traverse (Handle *h, visitproc visit, void *arg) {
 	return 0;
 }
 
+
+int Handle_check_initialized(Handle *handle, int expectedInitialized) {
+	if (!expectedInitialized == !handle->rk) {
+		return 1;  // OK
```

Comment:
Consider flipping 1 to 0 and change all places from `!Handle_check_initialized` to `Handle_check_initialized`